tests/GitWorkflowTest.php

```diff
@@ -504,4 +504,70 @@ public function executeCommand(string $command, array &$output = null, int &$ret
 		$messages = runGitWorkflow([$gitFile], $options, $shell, $debug);
 		$this->assertEquals($expected->getMessages(), $messages->getMessages());
 	}
+	function testFullGitWorkflowForInterBranchDiff() {
```

Comment:
Suuuuper minor, sorry 😓  but could we add a newline above this?

Reply:
Good catch! Fixed, thanks for the feedback.